qcodes_contrib_drivers/drivers/Ametek/SR_7270.py

```diff
@@ -0,0 +1,296 @@
+# This Python file uses the following encoding: utf-8
+
+"""
+Created by Elyjah <elyjah.kiyooka@cea.fr>, Jan 2022
+
+"""
+
+from qcodes import VisaInstrument, validators as vals
+from qcodes.utils.delaykeyboardinterrupt import DelayedKeyboardInterrupt
+from qcodes.utils.validators import ComplexNumbers
+
+class Signalrecovery7270(VisaInstrument):
+    """
+    This is the qcodes driver for the Ametex Signal Recovery
+    Model 7270 DSP Lockin amplifier
+
+    Note:
+    ask_raw command has been rewritten (bottom) to also read an echo to remove from buffer.
+    write_raw command have been rewritten to also read after writing using ask_raw.
+
+    """
+    def __init__(self, name: str, address: str, **kwargs):
+        super().__init__(name, address, terminator='', device_clear = True, **kwargs)
```

Comment:
this way the terminator can be changed without changing the driver code
```suggestion
    def __init__(self, name: str, address: str, terminator='', **kwargs):
        super().__init__(name, address, terminator=terminator, device_clear = True, **kwargs)
```

Reply:
thanks, good point.